frontend/src/metabase/search/components/DropdownSidebarFilter/DropdownSidebarFilter.tsx

```diff
@@ -144,14 +140,18 @@ export const DropdownSidebarFilter = ({
         autoWidth
         sizeToFit
       >
-        <SearchEventSandbox>
-          <Box w={popoverWidth ?? "100%"}>
-            <ContentComponent
-              value={selectedValues}
-              onChange={selected => onApplyFilter(selected)}
-            />
-          </Box>
-        </SearchEventSandbox>
+        {({ maxHeight }: { maxHeight: string }) => (
```

Comment:
I set `SearchEventSandbox` with `display: contents` since it had an unnecessary `div`, so I put the `maxHeight` on the `Stack` element that should prevent overflow within the popover container